server/mscalendar/views/calendar.go

```diff
@@ -20,22 +21,42 @@ func RenderCalendarView(events []*remote.Event, timeZone string) (string, error)
 		}
 	}
 
-	resp := "Times are shown in " + events[0].Start.TimeZone + "\n"
+	resp := "Times are shown in " + events[0].Start.TimeZone
 	for _, group := range groupEventsByDate(events) {
-		resp += group[0].Start.Time().Format("Monday January 02") + "\n"
+		resp += "\n" + group[0].Start.Time().Format("Monday January 02") + "\n\n"
+		resp += renderTableHeader()
 		for _, e := range group {
-			resp += fmt.Sprintf("* %s\n", renderEvent(e))
+			eventString, err := renderEvent(e, true)
+			if err != nil {
+				return "", err
+			}
+			resp += fmt.Sprintf("\n%s", eventString)
 		}
 	}
 
 	return resp, nil
 }
 
-func renderEvent(event *remote.Event) string {
+func renderTableHeader() string {
+	return `| Time | Subject |
+| :--: | :-- |`
+}
+
+func renderEvent(event *remote.Event, asRow bool) (string, error) {
 	start := event.Start.Time().Format(time.Kitchen)
 	end := event.End.Time().Format(time.Kitchen)
 
-	return fmt.Sprintf("%s - %s `%s`", start, end, event.Subject)
+	format := "(%s - %s) [%s](%s)"
+	if asRow {
+		format = "| %s - %s | [%s](%s) |"
```

Comment:
Can you post a screenshot of these two formats? I'm still curious about rendering the single event as a "headerless" table.

Reply:
Table format:
![Screenshot from 2020-04-24 15-48-04](https://user-images.githubusercontent.com/1933730/80219659-131fbf80-8643-11ea-92cb-3e24ba8a54db.png)
Single event:
![Screenshot from 2020-04-24 15-49-54](https://user-images.githubusercontent.com/1933730/80219870-5bd77880-8643-11ea-93df-2bd793e50bba.png)

Reply:
I honestly like the table more. It might look even better without the headers. @asaadmahmood Any thoughts on this?